src/Logger/LoggerInterface.php

```diff
@@ -5,23 +5,17 @@
 interface LoggerInterface
 {
     /**
-     * Method used by EppoClient to log assignments.
-     * Will pass to this method all the assignment information, and, based on implementation,
+     * Method used by EppoClient to log assignments to the data warehouse.
+     * This method will be passed all the assignment information, and, based on implementation,
      * try to log this information.
-     *
-     * @param string $experiment
-     * @param string $variation
-     * @param string $subject
-     * @param string $timestamp
-     * @param array $subjectAttributes
-     *
-     * @return void
      */
     public function logAssignment(
         string $experiment,
         string $variation,
         string $subject,
         string $timestamp,
-        array $subjectAttributes = []
+        array $subjectAttributes = [],
+        ?string $allocation = null,
+        ?string $featureFlag = null
```

Comment:
Unlike other SDKs that build an object that is then passed to the logger hook, we straight up send a bunch of parameters here. So I added the two new ones at the end and made them optional so as to be easily backward compatible.